docs/guide/moose.md

```diff
@@ -0,0 +1,42 @@
+## Clearpath Robotics' Moose
+
+%robot moose images/robots/moose/model.png
+
+The [Clearpath's Moose robot](https://www.clearpathrobotics.com/moose-ugv/) is a large all-terrain unmanned ground vehicle yet. It can handle tough environments with its rugged build, low ground pressure, and 8x8 traction tires, which allow effortless mobility through soft soils, vegetation, thick muds, and steep grades.
```

Comment:
```suggestion
The [Moose robot](https://www.clearpathrobotics.com/moose-ugv/) from [Clearpath Robotics](https://www.clearpathrobotics.com) is a large all-terrain unmanned ground vehicle yet.
It can handle tough environments with its rugged build, low ground pressure, and 8x8 traction tires, which allow effortless mobility through soft soils, vegetation, thick muds, and steep grades.
```